meson.build

```diff
@@ -192,7 +192,7 @@ install_data(
     install_dir: udev_dir / 'rules.d')
 
 package_datadir = join_paths('share', 'vpd')
-install_subdir('ibm_vpd/fru/', install_dir: package_datadir, strip_directory: true)
+install_subdir('config/ibm/', install_dir: package_datadir, strip_directory: true)
```

Comment:
we need config/vpd_inventory.json to be installed in /usr/share/vpd right?

Reply:
I saw it being there from other path "recipes-phosphor/vpd/ibm-vpd-parser/vpd_inventory.json", but yes I'll add that.

Reply:
I didn't mean to move vpd_inventory.json into config/ibm folder.
Let the vpd_inventory.json be in config/ directory.
While installing you install config/vpd_inventory.json as well.

Reply:
As discussed, let the vpd_inventory.json be in config/ibm directory itself. issue resolved